### DISCUSSION TITLE
Remaining Issues regarding PR#112

### DISCUSSION THREAD

**Discussion by lehors** [ossf/tac#118] [General]
PR#117 addresses many of the comments against PR#112 but some non trivial comments remain. Here is the list:

organizational-structure-overview.md:
Definition SIF - Default is to punt to later, merge as is ("TODO")
SIG - dlorenc and joshbressers want to leave out - Default is to merge as is.
jchestershopify suggests: SIG have "a very specific and terminable goal" - default is to merge as is.
At odd with current proposal for SIFs: there does not seem to be any such things as WG with SIFs and Project with SIFs, as opposed to just SIFs.
Brian proposes all SIGs and Projects to report to a WG (except for SIFs)
Brian: "Receives guidance on technical direction from TAC" seems very demanding of the TAC for Sandbox Projects

process/README:
Projects governance - implies WGs can create projects, but that's at odd with what the project liefycle - needs clarification - Brian suggested: WG's may launch their own SIGs without TAC approval, and of course in their own repo might have miscellaneous code or other artifacts to assist with their chartered mission, but if they want to launch a new Project (capital-P) then they need TAC approval.

process/request_resources.md:
Publication approval: TAC or no TAC? needs clarification, suggest: notification,  limit approval to "Technical Deliverables"
Remove "_Program TBD as of June 2022. Intended program details below._"?
Communications and content: Marketing Committee not critial-path for publishind on blog and newsletter.
Brian: remove Budget Committee, Maintainer stipends, Additional requests sections

process/project-lifecycle.md:
lukehinds: Allow for per-Project CoC. Brian: too complicated
"With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF" does not apply to SIFs - several instances
Brian would prefer list of "gets" to be limited to hard commits for Sandbox and Incubating Projects.
What does "or be initial code needed for an OpenSSF WG work" mean?
Brian: would benefit from more specific metrics re: "Maintain a diversified contributor base (i.e. not a single-vendor project)"
Brian: no entitlement to blog re: "May post project updates and tutorials to the OpenSSF blog"
Brian: strike out "maintainer stipends"

**Comment by brianbehlendorf** [comment]
Thank you for aggregating this all together! A mountain of hard work.

> "Brian proposes all WGs and Projects to report to a WG (except for SIFs)"

First use of "WGs" in that sentence should be "SIGs".
Also, I'm fine if a SIF also reports into a WG rather than directly to the TAC.

**Reply by lehors** [reply]
> Thank you for aggregating this all together! A mountain of hard work.
> 
> > "Brian proposes all WGs and Projects to report to a WG (except for SIFs)"
> 
> First use of "WGs" in that sentence should be "SIGs". Also, I'm fine if a SIF also reports into a WG rather than directly to the TAC.

You're right! Fixed.

**Comment by lehors** [comment]
PR merged
